tests/regression/run_TA_API.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
+
+# Build options
+OPTIONS=(
+    "|"
+    "|--proxy_passthrough"
+#   "--define mqtt=enable|"
```

Comment:
iri_host, iri_port, ta_host, ta_port, db_host, db_port, verbose should be added here

Reply:
The filename can be renamed to `run-api.sh`